easybuild/easyblocks/i/ipp.py

```diff
@@ -73,18 +83,27 @@ def install_step(self):
             }
 
         # in case of IPP 9.x, we have to specify ARCH_SELECTED in silent.cfg
-        if LooseVersion(self.version) >= LooseVersion('9.0'):
+        if LooseVersion(self.version) >= LooseVersion('9.0') and LooseVersion(self.version) < LooseVersion('2021.2.0'):
             silent_cfg_extras = {
                 'ARCH_SELECTED': self.arch.upper()
             }
 
+        """If installing from OneAPI, install only Intel IPP component"""
+        if LooseVersion(self.version) >= LooseVersion('2021.2.0'):
+            self.install_components = ['intel.oneapi.lin.ipp.devel']
+
         super(EB_ipp, self).install_step(silent_cfg_names_map=silent_cfg_names_map, silent_cfg_extras=silent_cfg_extras)
 
     def sanity_check_step(self):
         """Custom sanity check paths for IPP."""
         shlib_ext = get_shared_lib_ext()
 
-        dirs = [os.path.join('ipp', x) for x in ['bin', 'include', os.path.join('tools', 'intel64')]]
+        if LooseVersion(self.version) < LooseVersion('2021.2.0'):
+            dirs = [os.path.join('ipp', x) for x in ['bin', 'include', os.path.join('tools', 'intel64')]]
+        else:
+            dirs = [os.path.join('ipp/{}'.format(self.version), x)
```

Comment:
Do not use "xxx/", let os.path.join do its job. So change to 
```
os.path.join('ipp', '{}'.format(self.version), x)

Reply:
fixed